From 96b32c6f9ac66a8364471de5e809812ac1dcf835 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Thu, 27 Feb 2020 16:13:06 +0000
Subject: [PATCH 364/462] Mantis 0056312: Extend Desko-Penta Device-Handler by
 Accufill-SDK for reading US driving-licenses

add environment variables XSD_DIR, INC_XSD, XSD_COMP


git-svn-id: svn://localhost/SelfServiceBuild/trunk@670 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/scripts/setenv.bat | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/SelfServiceBuild/Build/scripts/setenv.bat b/SelfServiceBuild/Build/scripts/setenv.bat
index 2222f82..26d9285 100644
--- a/SelfServiceBuild/Build/scripts/setenv.bat
+++ b/SelfServiceBuild/Build/scripts/setenv.bat
@@ -405,6 +405,11 @@ set LIB_QTSDK=%DIR_QTSDK%\libraries\qtmlclient.lib
 set DIR_SED=%DIR_GLOBALSLIB%\sed\0.9.9.9\win32
 set SED=%DIR_SED%\bin\sed.exe
 
+set XSD_DIR=%DIR_GLOBALSLIB%\xsd\4.0.0\win
+set INC_XSD=-I%XSD_DIR%\libxsd
+set XSD_COMP=%XSD_DIR%\bin\xsd.exe
+
+
 if "%USE_SPRIT%"=="1.6.1" (
   goto USE_SPRIT_1.6.1
 ) else if "%USE_SPRIT%"=="BOOST" (
-- 
2.41.0.windows.1

